mod/discussions/start.php

```diff
@@ -16,6 +22,12 @@ function discussion_init() {
 
 	elgg_register_plugin_hook_handler('entity:url', 'object', 'discussion_set_topic_url');
 
+	// control context of discussions
+	elgg_register_plugin_hook_handler('container_permissions_check', 'object', 'discussion_topic_container_permissions', 700);
+
+	// allow group context
+	elgg_register_plugin_hook_handler('discussions:allow_context', 'group:', 'Elgg\Values::getTrue');
```

Comment:
Looks like this broke discussions because container_permissions_check happens for owner_guid too. :(